pyproject.toml

```diff
@@ -0,0 +1,163 @@
+  "Operating System :: Microsoft :: MS-DOS",
+  "Operating System :: Microsoft :: Windows",
+  "Operating System :: Microsoft",
+  "Operating System :: POSIX :: BSD :: FreeBSD",
+  "Operating System :: POSIX :: BSD",
+  "Operating System :: POSIX :: Linux",
+  "Operating System :: POSIX :: SunOS/Solaris",
+  "Operating System :: POSIX",
+  "Operating System :: Unix",
+  "Programming Language :: Python :: 3 :: Only",
+  "Programming Language :: Python :: 3",
+  "Programming Language :: Python :: 3.10",
+  "Programming Language :: Python :: 3.11",
+  "Programming Language :: Python :: 3.12",
+  "Programming Language :: Python :: Implementation",
+  "Programming Language :: Python",
+  "Programming Language :: Unix Shell",
+  "Topic :: Software Development :: Libraries :: Python Modules",
+  "Topic :: Software Development :: Libraries",
+  "Topic :: Software Development :: User Interfaces",
+  "Topic :: System :: Installation/Setup",
+  "Topic :: System :: Shells",
+  "Topic :: Terminals",
+  "Topic :: Utilities",
+  "Typing :: Typed",
+]
+dependencies = [
+  "clipdetect>=0.1.4",
+  "deepdiff>=6.5.0",
+  "anytree>=2.12.1",
+  "einops==0.5.0",
+  "g2p~=2.0",
+  "gradio>=4.32.1",
+  "grapheme>=0.6.0",
+  "ipatok>=0.4.1",
+  "librosa==0.9.2",
+  "lightning>=2.0.0",
+  "loguru==0.6.0",
+  "matplotlib~=3.9.0",
+  "merge-args",
+  "nltk==3.9.1",
+  "numpy<2",                       # torch < 2.4.1 requires numpy < 2 but fails to declare it
+  "pandas~=2.0",
+  "panphon==0.20.0",
+  "protobuf~=4.25",                # https://github.com/EveryVoiceTTS/EveryVoice/issues/387
+  "pycountry==22.3.5",
+  "pydantic[email]>=2.4.2,<2.8.0",
+  "pympi-ling",
+  "pysdtw==0.0.5",
+  "pyworld==0.3.4",
+  "PyYAML>=5.1",
+  "questionary==1.10.0",
+  "simple-term-menu==1.5.2",
+  "setuptools==59.5.0",            # https://github.com/pytorch/pytorch/issues/69894
+  "tabulate==0.9.0",
+  "tensorboard>=2.14.1",
+  "torch==2.1.0",
+  "torchaudio==2.1.0",
+  "torchinfo==1.8.0",
+  "typer>=0.12.3",
+  "yaspin>=3.1.0",
+]
+dynamic = ["version"]
+
+[project.optional-dependencies]
+# [Specifying GPU version of pytorch for python package in pyproject.toml](https://discuss.pytorch.org/t/specifying-gpu-version-of-pytorch-for-python-package-in-pyproject-toml/209157)
+torch = [
+  # these requirements have to be installed ahead of time in your environment and from a different URL:
+  # CUDA_TAG=cu118 pip install -r requirements.torch.txt --find-links https://download.pytorch.org/whl/torch_stable.html
+  'torch==2.1.0; sys_platform == "darwin"',
+  'torchaudio==2.1.0; sys_platform != "darwin"',
+]
+dev = [
+  "black~=24.3",
+  "flake8>=4.0.1",
+  "gitlint-core>=0.19.0",
+  "isort>=5.10.1",
+  "mypy>=1.8.0",
+  "pre-commit>=3.2.0",
+  "types-pyyaml>=6.0.5",
+  "types-requests>=2.27.11",
+  "types-setuptools>=57.4.9",
+  "types-tabulate==0.9.0",
+  "types-tqdm>=4.64,<5.0",
+  # [Self-referential](https://hatch.pypa.io/dev/config/dependency/#self-referential)
+  #-r requirements.test.txt
```

Comment:
I don't feel the need to leave this comment here, it's standard pyproject.toml syntax.